### ISSUE TITLE
Add gradle to supporting companies

### ISSUE THREAD

**Issue by atomfrede** [jhipster/generator-jhipster#27119]
##### **Overview of the feature request**

Add gradle as supporting company to the new section on the website. I know this section is intended to be for open collective sponsors, but I would say its fair to show gradle there too.

![image](https://github.com/user-attachments/assets/6008b31f-3296-43ea-8f80-889a85851cb7)

##### **Motivation for or Use Case**

They give us free develocity instance and license for both gradle and maven for free. 

- [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by DanielFran** [comment]
@atomfrede Already available here: https://www.jhipster.tech/contributing-companies/

**Comment by atomfrede** [comment]
@DanielFran Yes true, but I think the contributing companies is a little different imho. We should update it at least and make clear that gradle is supporting us with a paid service for free, so I would put them under supporting companies on the start page.
